api/src/test/java/org/openmrs/module/fhir/api/diagnosticreport/handler/LaboratoryHandlerTest.java

```diff
@@ -0,0 +1,255 @@
+package org.openmrs.module.fhir.api.diagnosticreport.handler;
+
+import org.hl7.fhir.dstu3.model.DiagnosticReport;
+import org.hl7.fhir.dstu3.model.Reference;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.Concept;
+import org.openmrs.Encounter;
+import org.openmrs.EncounterRole;
+import org.openmrs.EncounterType;
+import org.openmrs.Patient;
+import org.openmrs.PatientIdentifier;
+import org.openmrs.PatientIdentifierType;
+import org.openmrs.Person;
+import org.openmrs.PersonAddress;
+import org.openmrs.PersonName;
+import org.openmrs.Provider;
+import org.openmrs.Location;
+import org.openmrs.api.EncounterService;
+import org.openmrs.api.PatientService;
+import org.openmrs.api.context.Context;
+
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+import org.openmrs.module.fhir.api.diagnosticreport.handler.LaboratoryHandler;
+
+import java.util.Date;
+import java.util.Set;
+import java.util.List;
+import java.util.LinkedHashSet;;
+import java.util.TreeSet;;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+
+public class LaboratoryHandlerTest extends BaseModuleContextSensitiveTest {
+
+    private PatientService patientService;
+
+    private EncounterService encounterService;
+
+    /**
+     * Runs before every test and sets patientService and encounterService
+     */
+    @Before
+    public void runBeforeEveryTest() {
+        patientService = Context.getPatientService();
+        encounterService = Context.getEncounterService();
+    }
+
+    /**
+     * Creates and returns a Patient
+     * 
+     * @return PPatient object
+     */
+    private Patient createPatient()
```

Comment:
Properly format it as `private Encounter createEncounter(String patient_uuid) {`